doc/trapezoidal_industrial_planner/trapezoidal_industrial_planner.rst

```diff
@@ -0,0 +1,371 @@
+Trapezoidal Industrial Planner
+==============================
+
+``trapezoidal_industrial_planner`` provides a trajectory generator to plan standard robot
+motions like PTP, LIN, CIRC with the interface of a MoveIt PlannerManager
+plugin.
+
+User Interface MoveGroup
+------------------------
+
+This package implements the ``planning_interface::PlannerManager``
+interface of MoveIt. By loading the corresponding planning pipeline
+(``trapezoidal_planning_pipeline.launch.xml`` in your
+``*_moveit_config`` package), the trajectory generation
+functionalities can be accessed through the user interface (c++, python
+or rviz) provided by the ``move_group`` node, e.g.
+``/plan_kinematics_path`` service and ``/move_group`` action. For
+detailed usage tutorials please refer to :doc:`/doc/move_group_interface/move_group_interface_tutorial`.
+
+Joint Limits
+------------
+
+For all commands the planner needs to know the limitations of each robot
+joint. The limits used are calculated from the limits set in the urdf as
+well as the limits set on the parameter server (usually put there by
```

Comment:
I would note that the file is generated by the setup assistant and the average user does not need to worry about it.

Reply:
I've rephrased the paragraph; is it better now?